apps/daimo-mobile/src/action/useNFCReader.ts

```diff
@@ -0,0 +1,76 @@
+import { parseDaimoLink } from "@daimo/common";
+import { useEffect } from "react";
+import NfcManager, { NfcTech } from "react-native-nfc-manager";
+import { Hex, hexToBytes } from "viem";
+
+import { handleDeepLink, useNav } from "../view/shared/nav";
+
+let isReadingNfc = false;
+
+export function useNFCReader() {
+  const nav = useNav();
+  useEffect(() => {
+    console.log("[NFC] init");
+    initNFC(nav);
+  }, []);
+}
+
+async function initNFC(nav: ReturnType<typeof useNav>) {
+  await NfcManager.start();
+  console.log("WTF");
+  const isEnabled = await NfcManager.isEnabled();
+  const isSupported = await NfcManager.isSupported();
+  console.log(`[NFC] isEnabled: ${isEnabled}, isSupported: ${isSupported}`);
+  if (!isEnabled || !isSupported) return;
+
+  setInterval(() => {
+    if (isReadingNfc) return;
+    readNdef(nav);
+  }, 1000);
```

Comment:
can this be made much lower, like 100? atm I think this means that the expected time between being in range of the other device and getting the ding is at least half a second which feels like a lot

Reply:
from @markusbug : this is just the reset interval after a successful read